websurfx/config.lua

```diff
@@ -43,11 +43,20 @@ safe_search = 2
 -- tomorrow-night
 -- }}
 colorscheme = "catppuccin-mocha" -- the colorscheme name which should be used for the website theme
-theme = "simple"                 -- the theme name which should be used for the website
+-- The different themes provided are:
+-- {{
+-- simple
+-- }}
+theme = "simple" -- the theme name which should be used for the website
+-- The different animations provided are:
+-- {{
+-- simple-frost
```

Comment:
```suggestion
-- simple-frosted-glow
```
Shouldn't this be `simple-frosted-glow`

Reply:
Thanks :heart: for reviewing and pointing out this mistake. I will make the change according to your suggestion right away :rocket: :slightly_smiling_face: